app/Http/Requests/CreateUserRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
```

Comment:
có hiểu hàm nay có tác dụng gì không?

Reply:
xem ai được thực hiện request này.

Reply:
vậy sao c toàn trả về true?, k có điều kiện check à?

Reply:
ko có điều kiên gì cả. để validate thôi mà?